pkg/virt-handler/hotplug-disk/mount_test.go

```diff
@@ -248,8 +250,11 @@ var _ = Describe("HotplugVolume block devices", func() {
 
 	It("mountBlockHotplugVolume and unmountBlockHotplugVolumes should make appropriate calls", func() {
 		blockSourcePodUID := types.UID("fghij")
-		hotplugdisk.SetKubeletPodsDirectory(tempDir)
-		targetPodPath := filepath.Join(tempDir, string(m.findVirtlauncherUID(vmi)), "volumes/kubernetes.io~empty-dir/hotplug-disks")
+		mknodCommand = func(deviceName string, major, minor int64, blockDevicePermissions string) ([]byte, error) {
```

Comment:
This function appears several times in this test module. Is there a way to reduce copy/paste?

Clever use of MkdirAll by the way.

Reply:
Most of them are slightly different. I am not sure if it is great usage. I think that needs a bigger rewrite later.